### PR TITLE
Feature/compute

### PR DESCRIPTION
**Pull request by alexcos20**
Add Compute Service Endpoints

### CODE REVIEW

**Comment by kremalicious** [comment]
This PR does only define the operator endpoints, yet the compute feature spans across multiple projects. If this PR is supposed to only define operator service endpoints, then that's fine.

There should be one document defining the compute services across repos. This document is https://github.com/oceanprotocol/dev-ocean/blob/master/doc/architecture/computing-services.md which hasn't been updated since 10 months, and the specific endpoints could be outlined in there too. So if we add more docs regarding compute, then at least they should be linked to from that overall doc. My point being we should maintain our docs properly, otherwise we will end up with a bunch of disconnected and outdated docs

File: doc/architecture/compute/operator-service-endpoints.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  
+  
+### GET /api/v1/operator/status
+   
+   
+Get all workflows and corresponding stats
+
+Parameters
+```
+        owner: String object containg user ETH address (optional)
```
Comment (kremalicious) [inline doc/architecture/compute/operator-service-endpoints.md]: why is everything optional? What happens then if I pass no parameter?

File: doc/architecture/compute/operator-service-endpoints.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        agreementID: String object containing agreementID (optional)
+        workflowID: String object containing workflowID (optional)
+```
+
+Returns
+
+An Array of objects, each object describing a workflow. If the array is empty, then the search yeld no results
```
Comment (kremalicious) [inline doc/architecture/compute/operator-service-endpoints.md]: so I pass in a unique workflow ID, but get multiple workflows back? Isn't every workflow supposed to have one unique ID?

Comment (alexcos20) [reply doc/architecture/compute/operator-service-endpoints.md]: No, you will only get one workflow.
But if you search using only ownerID, you will get multiple workflows if user has multiple jobs

File: doc/architecture/compute/operator-service-endpoints.md (hunk 1 above)
Comment (alexcos20) [reply doc/architecture/compute/operator-service-endpoints.md]: Fixed. 
At least one input is required (any of them)

File: doc/architecture/compute/operator-service-endpoints.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+Output:
+```
+[
+      {
+        "owner":"0x1111",
+        "agreementID":"0x2222",
+        "workflowID":"3333",
+        "createdat":"2020-10-01T01:00:00Z",
+        "finishedat":"2020-10-01T01:00:00Z",
+        "status":5,
```
Comment (kremalicious) [inline doc/architecture/compute/operator-service-endpoints.md]: users shouldn't be required to consult a table to map numbers to text. So status should be passed as text too, like `statusText`

File: doc/architecture/compute/operator-service-endpoints.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+```
+
+Output:
+```
+[
+      {
+        "owner":"0x1111",
+        "agreementID":"0x2222",
+        "workflowID":"3333",
+        "createdat":"2020-10-01T01:00:00Z",
```
Comment (kremalicious) [inline doc/architecture/compute/operator-service-endpoints.md]:
```suggestion
        "dateCreated":"2020-10-01T01:00:00Z",
```

File: doc/architecture/compute/operator-service-endpoints.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+        "finishedat":"2020-10-01T01:00:00Z",
```
Comment (kremalicious) [inline doc/architecture/compute/operator-service-endpoints.md]:
```suggestion
        "dateFinished":"2020-10-01T01:00:00Z",
```

File: doc/architecture/compute/operator-service-endpoints.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+            "http://example.net/logs/output/0",
+            "http://example.net/logs/output/1"
+            }
+         ]
+       },
+       {
+        "owner":"0x1111",
+        "agreementID":"0x2222",
+        "workflowID":"3333",
+        "createdat":"2020-10-01T01:00:00Z",
```
Comment (kremalicious) [inline doc/architecture/compute/operator-service-endpoints.md]:
```suggestion
        "dateCreated":"2020-10-01T01:00:00Z",
```

File: doc/architecture/compute/operator-service-endpoints.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+        "finishedat":"2020-10-01T01:00:00Z",
```
Comment (kremalicious) [inline doc/architecture/compute/operator-service-endpoints.md]:
```suggestion
        "dateFinished":"2020-10-01T01:00:00Z",
```

File: doc/architecture/compute/operator-service-endpoints.md (hunk 3 above)
Comment (alexcos20) [reply doc/architecture/compute/operator-service-endpoints.md]: fixed

File: doc/architecture/compute/operator-service-endpoints.md [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+```
+GET /api/v1/operator/status?owner=0x1111
+```
+
+Output:
+```
+[
+      {
+        "owner":"0x1111",
+        "agreementID":"0x2222",
```
Comment (kremalicious) [inline doc/architecture/compute/operator-service-endpoints.md]: we do pascalCase, right? So would be `agreementId`

**Comment by ssallam** [comment]
The operator service endpoints have been moved to the operator-service repo.

**Pull request closed without merging**